src/mozanalysis/experiment.py

```diff
@@ -367,17 +393,35 @@ def build_enrollments_query(
             time_limits, enrollments_query_type
         )
 
+        if exposure_metric:
+            exposure_query = (
+                custom_exposure_query
```

Comment:
Is this branch possible?

Reply:
Yeah. It is possible to specify both `exposure_metric` and `custom_exposure_query` in `get_time_series_data()`. However, `custom_exposure_query` will take precedence